test/e2e/e2e.go

```diff
@@ -1282,6 +1269,9 @@ var _ = ginkgo.Describe("e2e non-vxlan external gateway and update validation",
 		if err != nil {
 			framework.Failf("failed to add the pod host route on the test container: %v", err)
 		}
+		_, err = runCommand("docker", "exec", gwContainerNameAlt1, "ping", "-c", "5", pingSrc)
```

Comment:
we are moving the ping around so that the ARP entry will show up in the exgw pod. I think in the future we should add static arp entries for the nodes in the exgw and revert moving this stuff around. Its ok for now though...